### ISSUE TITLE
Add other collision evaluators to trajopt_ifopt

### ISSUE THREAD

**Issue by mpowelson** [ros-industrial-consortium/trajopt_ros#193]

**Comment by mpowelson** [comment]
@steviedale is tackling this one.

**Comment by mpowelson** [comment]
@Levi-Armstrong @colin-lewis-19 Has any progress been made on this one? Ideally we would also update this to use the tesseract_collision::CollisionCheckConfig before [Tesseract #443](https://github.com/ros-industrial-consortium/tesseract/pull/443)

**Comment by Levi-Armstrong** [comment]
I don't think so. I took a look a week ago to see what it would take but I did not start.

**Comment by mpowelson** [comment]
The final thing to do here is to allow the constraint to take any of these. Currently only discrete is allowed
